src/sdk/SpeechTranslationConfig.ts

```diff
@@ -87,8 +92,8 @@ export abstract class SpeechTranslationConfig extends SpeechConfig {
 
     /**
      * Gets/Sets the authorization token.
-     * If this is set, subscription key is ignored.
-     * User needs to make sure the provided authorization token is valid and not expired.
+     * Note: The caller needs to ensure that the authorization token is valid. Before the authorization token
+     * expires, the caller needs to refresh it by calling this setter with a new valid token.
```

Comment:
>calling this setter [](start = 50, length = 19)

setting this property? there's no setter

Reply:
SpeechTranslationConfig extends SpeechConfig and speechconfig has a getter.